modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -112,148 +48,74 @@ metadata:
 ----
 
-+
-. Start the virtual machine. If the virtual machine is already running, restart it.
-+
-
-+
-. Query the `Service` object to verify it is available and is configured with type `ClusterIP`.
-+
+. Start the VM. If the VM is already running, restart it.
 
 .Verification
-* Run the `oc get service` command, specifying the `namespace` that you reference in the virtual machine and `Service` YAML files.
+. Query the `Service` object to verify that it is available:
 +
-
 [source, terminal]
 ----
 $ oc get service -n example-namespace
 ----
 +
-
-.Example output
+.Example output for `ClusterIP` service
 [source, terminal]
 ----
 NAME        TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
 vmservice   ClusterIP   172.30.3.149   <none>        27017/TCP   2m
 ----
 +
-
-** As shown from the output, `vmservice` is running.
-** The `TYPE` displays as `ClusterIP`, as you specified in the `Service` YAML.
-
-. Establish a connection to the virtual machine that you want to use to back your service. Connect from an object inside the cluster, such as another virtual machine.
-+
-
-.. Edit the virtual machine YAML as follows:
-+
-
-[source,yaml]
+.Example output for `NodePort` service
+[source, terminal]
 ----
-apiVersion: kubevirt.io/v1
-kind: VirtualMachine
-metadata:
-  name: vm-connect
-  namespace: example-namespace
-spec:
-  running: false
-  template:
-    spec:
-      domain:
-        devices:
-          disks:
-            - name: containerdisk
-              disk:
-                bus: virtio
-            - name: cloudinitdisk
-              disk:
-                bus: virtio
-          interfaces:
-          - masquerade: {}
-            name: default
-        resources:
-          requests:
-            memory: 1024M
-      networks:
-        - name: default
-          pod: {}
-      volumes:
-        - name: containerdisk
-          containerDisk:
-            image: kubevirt/fedora-cloud-container-disk-demo
-        - name: cloudinitdisk
-          cloudInitNoCloud:
-            userData: |
-              #cloud-config
-              user: fedora
-              password: fedora
-              chpasswd: {expire: False}
-# ...
+NAME        TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)            AGE
+vmservice   NodePort    172.30.232.73   <none>       27017:30000/TCP    5m
 ----
 +
-
-.. Run the `oc create` command to create a second virtual machine, where `file.yaml` is the name of the virtual machine YAML:
-+
-
-[source,terminal]
+.Example output for `LoadBalancer` service
+[source, terminal]
 ----
-$ oc create -f <file.yaml>
+NAME        TYPE            CLUSTER-IP     EXTERNAL-IP                    PORT(S)           AGE
+vmservice   LoadBalancer    172.30.27.5   172.29.10.235,172.29.10.235     27017:31829/TCP   5s
 ----
-+
-
-.. Start the virtual machine.
 
-.. Connect to the virtual machine by running the following `virtctl` command:
+. Choose the appropriate method to connect to the virtual machine:
```

Comment:
Re: this last step's sub-steps:
- Should they have articles? "For a `ClusterIP` service" (etc.)?
- My instinct is to say that this should be a bulleted list rather than a/b/c, since you aren't necessarily going to do them all (or do them in order)